### PR TITLE
LPS-132723

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
FWD: https://github.com/liferay-data-engine/liferay-portal/pull/571
I was able to fix the issue and forwarding the new fix.

https://issues.liferay.com/browse/LPS-132723

It can be difficult for the user to use the web content toolbar options if the text is very long. This was verified as a bug in this comment: https://issues.liferay.com/browse/PTR-2478?focusedCommentId=2462821&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2462821

This is resolved by having "resize_enabled", true and disabling autogrow plugin for the editor. In order to do this, I had to create JournalArticleContentEditorConfigContributor to edit the Web Content and edit JournalArticleDescriptionEditorConfigContributor. I was able to check this change resolves the issue.

Let me know if there are any questions or comments about this.
Thank you.

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
Just started reviewing.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java [hunk 1]
```diff
@@ -73,9 +73,19 @@
 		HttpServletRequest httpServletRequest =
 			ddmFormFieldRenderingContext.getHttpServletRequest();
 
+		String portletNamespace =
+			ddmFormFieldRenderingContext.getPortletNamespace();
+
+		String portletName = StringPool.BLANK;
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java:79]: Every request has the `portletId` information that you're trying to extract from the `portletNamespace`. Could you please use the `ParamUtil` to get it, instead using `substring`? I believe you can work this one out by directly calling the following line in the `getEditorConfiguration` parameter:
`ParamUtil.getString(PortalUtil.getOriginalServletRequest(ddmFormFieldRenderingContext.getHttpServletRequest()), "p_p_id")`

Thx.

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hi, @fortunatomaldonado. Thanks for contributing.

Could you please check the notes I've made down below?

Kind regards.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java]: @rodrigopaulino, thank you for this information! I have made the changes and will rerun the tests.
Thanks again!

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java]: @rodrigopaulino It looks like the SF does not want to use `PortalUtil`. I will try to use portal service instead.

**Comment by rodrigopaulino** [comment]
@fortunatomaldonado, I guess the SF is requesting you to use the @Reference to the Portal service, since you're inside an OSGi component. Could you please do that? Thx.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java]: Correct. Thanks! 😄

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
@rodrigopaulino, made the changes. Thank you for all the help!

**Review by rodrigopaulino — APPROVED** [review]
LGTM

**Comment by rodrigopaulino** [comment]
@fortunatomaldonado, noticed that you run relevant two times before stopping one of them. Please wait for the first attempt's results, then run it again if it's necessary.

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**